app/src/main/java/org/wikipedia/diff/ArticleEditDetailsFragment.kt

```diff
@@ -349,23 +363,28 @@ class ArticleEditDetailsFragment : Fragment(), WatchlistExpiryDialog.Callback, L
     override fun onMenuItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.menu_share_edit -> {
+                PatrollerExperienceEvent.logAction("top_menu_share_click", "pt_diff_view")
```

Comment:
Not sure but should we add logic to `PatrollerExperienceEvent` `logAction` where to only limit to log the action if the source was from the recent edits list? We can use `viewModel.fromRecentEdits` in the logic.